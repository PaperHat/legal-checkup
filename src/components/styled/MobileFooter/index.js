import styled from 'styled-components'

const MobileFooter = styled.div`
  height: 130px;
  text-align: center;
  display: flex;
  flex-direction: column;
  justify-content: center;
  background-color: #202020;
`
MobileFooter.displayName = 'StyledMobileFooter'

export default MobileFooter
