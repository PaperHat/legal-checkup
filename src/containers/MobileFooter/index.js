import {
  PRIVACY_POLICY_PATH,
  TERMS_OF_USE_PATH,
  COMMUNITY_LAW_PATH
} from '@state/constants'

import StyledMobileFooter from '@components/styled/MobileFooter'
import LinkExternal from '@components/styled/LinkExternal'
import MobileFooterLink from '../MobileFooterLink'
import React from 'react'

export default function Footer () {
  return (
    <StyledMobileFooter>
      <MobileFooterLink href={COMMUNITY_LAW_PATH} target='_blank'>
        Â© 2018 Community Law
      </MobileFooterLink>
      <MobileFooterLink to={PRIVACY_POLICY_PATH}>
        Privacy Policy
      </MobileFooterLink>
      <MobileFooterLink to={TERMS_OF_USE_PATH}>Terms of Use</MobileFooterLink>
    </StyledMobileFooter>
  )
}
