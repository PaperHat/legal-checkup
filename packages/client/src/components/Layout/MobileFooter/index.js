import { PRIVACY_POLICY_PATH, TERMS_OF_USE_PATH, COMMUNITY_LAW_PATH } from '../../../state/constants'

import React from 'react'
import styled from 'styled-components'
import MobileFooterLink from '../../Link/MobileFooterLink'

const StyledMobileFooter = styled.div`
  height: 130px;
  text-align: center;
  display: flex;
  flex-direction: column;
  justify-content: center;
  background-color: #202020;
`
StyledMobileFooter.displayName = 'StyledMobileFooter'

export default function MobileFooter () {
  return (
    <StyledMobileFooter>
      <MobileFooterLink href={COMMUNITY_LAW_PATH} target='_blank'>
        2018 by Community Law Wellington and Hutt Valley
      </MobileFooterLink>
      <MobileFooterLink to={PRIVACY_POLICY_PATH}>Privacy Policy</MobileFooterLink>
      <MobileFooterLink to={TERMS_OF_USE_PATH}>Terms of Use</MobileFooterLink>
    </StyledMobileFooter>
  )
}
