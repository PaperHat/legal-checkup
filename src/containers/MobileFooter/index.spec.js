import React from 'react'
import { shallow } from 'enzyme'

import MobileFooter from '.'

describe('components:MobileFooter', function () {
  it('renders the MobileFooter and CSS properties properly', () =>
    expect(
      toJson(shallow(<MobileFooter>MobileFooter</MobileFooter>))
    ).toMatchSnapshot())
})
