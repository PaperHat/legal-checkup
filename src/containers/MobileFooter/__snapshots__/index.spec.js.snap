// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:MobileFooter renders the MobileFooter and CSS properties properly 1`] = `
<StyledMobileFooter>
  <StyledLinkExternalMobile
    href="http://communitylaw.org.nz/"
    target="_blank"
  >
    Â© 2018 Community Law
  </StyledLinkExternalMobile>
  <Connect(StyledMobileFooterLink)
    to="/privacy-policy"
  >
    Privacy Policy
  </Connect(StyledMobileFooterLink)>
  <Connect(StyledMobileFooterLink)
    to="/terms-of-use"
  >
    Terms of Use
  </Connect(StyledMobileFooterLink)>
</StyledMobileFooter>
`;
